fe/fe-core/src/main/java/com/starrocks/server/StorageVolumeMgr.java

```diff
@@ -55,6 +64,12 @@ public abstract class StorageVolumeMgr {
     @SerializedName("storageVolumeToTables")
     protected Map<String, Set<Long>> storageVolumeToTables = new HashMap<>();
 
+    @SerializedName("dbToStorageVolumes")
+    protected Map<Long, String> dbToStorageVolume = new HashMap<>();
```

Comment:
`dbToStorageVolume` and `tableToStorageVolume` can be recovered from `storageVolumeToDbs` and `storageVolumeToTables`, no need to serialize.